istioctl/cmd/authz.go

```diff
@@ -157,14 +145,16 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 	}
 
 	validatorCmd = &cobra.Command{
-		Use:   "validate <policy-file1,policy-file2,...>",
-		Short: "Validate authentication and authorization policy",
-		Long: `This command goes through all authorization policy files and finds potential issues such as:
-						* ServiceRoleBinding refers to a non existing ServiceRole.
-						* ServiceRole not used.
-           It does not require access to the cluster as the validation is against local files.
-					`,
-		Example: "istioctl experimental auth validate -f policy1.yaml,policy2.yaml",
+		Use:   "validate v1alpha1-policy.yaml",
```

Comment:
This tool is never used and we're deprecating v1alpha1 soon. How about we delete this dead code in another PR to avoid the burden of maintaining it?

Reply:
Yeah I was thinking the same thing. It also does very limited checking and could be merged with the top level analyze command.